src/Entities/Frames.jl

```diff
@@ -49,8 +50,55 @@ end
 
 local_dof_symbol(::Frame) = [:u, :θ]
 
-function internal_forces(m::AbstractHyperElasticMaterial, f::Frame, u_e::AbstractVector)
-    error("Not implemented.")
-end
+function internal_forces(m::IsotropicLinearElastic, f::Frame, u_e::AbstractVector)
+    @assert f.mass_matrix == Consistent
+
+    # Reference:
+    # (ux1, uy1, uz1, ux2, uy2, uz2, θx1, θy1, θz1, θx2, θy2, θz2) = u_e
+
+    # TODO Generalize.
```

Comment:
```suggestion
    # TODO Generalize.
```
This is now reflected in #393

Reply:
I added the issue number instead of removing.